src/main/kotlin/view/MainView.kt

```diff
@@ -52,34 +54,49 @@ fun DisplayAlgorithmMenu(name: String) {
                 ImageButton(
                     imageResourceId = image,
                     onClick = {
-                    }
+                        if(image == "FindBridge.svg"){
+                            onClick()
+                            }
+                    }, viewModel
```

Comment:
так как по умолчанию у нас пустная функция `{}`, полагаю, можно сделать просто вот так:

```suggestion
                ImageButton(imageResourceId = image, onClick = onClick, viewModel)
```